homeassistant/components/bring/strings.json

```diff
@@ -26,6 +26,44 @@
       "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]"
     }
   },
+  "entity": {
+    "sensor": {
+      "urgent": {
+        "name": "Urgent"
+      },
+      "convenient": {
+        "name": "On occasion"
+      },
+      "discounted": {
+        "name": "Discount only"
```

Comment:
Im missing what these entities represent

Reply:
The Bring App has since the last update badges that you can assign to an item on the list. The sensors will show if, and how many items were tagged with one of these badges. 
![image](https://github.com/user-attachments/assets/2235d1ed-6807-46ce-9242-6661258f7b5d)

Urgent means, well, need them ASAP, and the opposite is on occassion. Discount only means to buy those items only if they are on discount. (Although every family could have a different definition for those badges and how they use them)

I can imagine some automations, like send a notification and add a to-do `Go shopping 🛒` with a due date for today
![There'SLiterallyEndlessPossibilitiesSamusPaulicelliGIF](https://github.com/user-attachments/assets/d9e5ee1b-4ef9-4a09-8e57-1a64906e96b7)

Reply:
Oh but that is fine, but these sensors have no unit of measurement and their name also didn't give any context. I would asume urgent meant this, but it could be made clearer

Reply:
I couldn't come up with a name, that would make it clear but still be short enough. The description in the docs should eliminate clarifications^^